src/notification.cpp

```diff
@@ -653,8 +906,13 @@ void Notification::close()
 /*!
     \qmlsignal Notification::clicked()
 
-    Sent when the notification is clicked (its default action is invoked).
```

Comment:
Should this have a note to encourage use of remote actions rather than onClicked?